src/editor.c

```diff
@@ -689,8 +685,11 @@ static gboolean reshow_calltip(gpointer data)
 }
 
 
-static void request_reshowing_calltip(SCNotification *nt)
+static void request_reshowing_calltip(GeanyDocument *doc, SCNotification *nt)
 {
+	if (plugin_extension_calltips_provided(doc, NULL))
+		return;
+
```

Comment:
I don't really care, but I'd have put the test at the single call site rather than adding a parameter only for returning early

Reply:
Just to be sure, you mean inside `case SCN_AUTOCCANCELLED:` below, right?

Reply:
Yes